torch/_torch_docs.py

```diff
@@ -3892,6 +3892,45 @@ def merge_dicts(*dicts):
     tensor([ 0.,  2.,  1.,  0.])
 """.format(**common_args))
 
+add_docstr(torch.histogram,
+           r"""
+histogram(input, bins, *, range=None, weight=None, density=False, out=None) -> (Tensor, Tensor)
+
+Computes the histogram of the values in a tensor.
+
+If :attr:`bins` is an int, it specifies the number of equal-width bins.
+:attr:`range` may be used to specify the lower and upper range of the bins.
+If no range is specified, the minimum and maximum elements of the input
+tensor are used.
+
+If :attr:`bins` is a tensor, it specifies the sequence of bin edges.
+
+Args:
+    {input}
+    bins: int or 1D Tensor. If int, defines the number of equal-width bins. If tensor,
+          defines the sequence of bin edges including the rightmost edge.
+
+Keyword args:
+    range (tuple of float): Defines the range of the bins.
+    weight (Tensor): If provided, weight should have the same shape as input. Each value in
+                     input contributes its associated weight towards its bin's result.
+    density (bool): If False, the result will contain the count (or total weight) in each bin.
+                    If True, the result is the value of the probability density function over the bins,
```

Comment:
I don't think we should change this, but the mathematician in me wants to be clear about what we're taking a pdf of. That is, I think a more precise description would be that this returns the values of the (piecewise-constant) pdf of the counting measure defined by the original histogram values.